websites/J/JoJo's Bizarre Encyclopedia/presence.ts

```diff
@@ -2,10 +2,15 @@ const presence = new Presence({
 	clientId: "1007688410875564062",
 });
 
-presence.on("UpdateData", () => {
+presence.on("UpdateData", async () => {
 	let presenceData: PresenceData = {};
 
-	if (window.location.href.toLowerCase().includes("jojowiki.com")) {
+	const showButtons = await presence.getSetting("showButtons"),
+		{ hostname, pathname, origin, href, searchParams } = new URL(
+			window.location.href
+		);
```

Comment:
Sorry, instead you may just do this instead 
```suggestion
		{ hostname, pathname, origin, href, search } = document.location,
		searchParams = new URLSearchParams(search)
```